src/app/features/auth/pages/login/login-page.component.ts

```diff
@@ -36,9 +36,12 @@ export class LoginPageComponent {
       this._store.dispatch(new SetIsLoading(false))
       if (err) {
         throw new Error('Login failed')
-        return
       }
       void this._router.navigate(['/'])
     }, 1000)
   }
+
+  public goToCreateAccount(): void {
+    void this._router.navigate(['auth/register'])
+  }
```

Comment:
Why is it both "Create Account" and "Register" event in the same place? Be consistent when naming things.